Drive/rover_drive_system.hpp

```diff
@@ -17,6 +17,8 @@ class RoverDriveSystem
  public:
   struct MissionControlData
   {
+    int heartbeatCount;
+    int serverHeartbeat;
```

Comment:
remove serverHeartbeat and just use heartbeatCount variable. This way there will be a hearbeatCount member variable and a heartbeatCount variable within the MissionControlData  data structure